notes/api/auth/clients.sh

```diff
@@ -0,0 +1,58 @@
+# notes/docker.sh           Start docker compose
+# notes/postgres.sh         Reset database
+
+INSTANCE=api/auth/clients
+DATASET=$INSTANCE
+# notes/spinta/server.sh    Configure server
+
+cat > $BASEDIR/manifest.txt <<EOF
+d | r | b | m | property | type   | access
+$DATASET                 |        | 
+  |   |   | City         |        | 
+  |   |   |   | name     | string | open
+EOF
+poetry run spinta copy $BASEDIR/manifest.txt -o $BASEDIR/manifest.csv
+cat $BASEDIR/manifest.csv
+poetry run spinta show $BASEDIR/manifest.csv
+
+# notes/spinta/server.sh    Check configuration
+# notes/spinta/server.sh    Run migrations
+# notes/spinta/server.sh    Add client
+# notes/spinta/server.sh    Run server
+
+# Configure client
+SERVER=:8000
+CLIENT=test
+SECRET=secret
+SCOPES=(
+    spinta_set_meta_fields
+    spinta_getone
+    spinta_getall
+    spinta_search
+    spinta_changes
+    spinta_insert
+    spinta_upsert
+    spinta_update
+    spinta_patch
+    spinta_delete
+    spinta_wipe
+    spinta_auth_clients
+)
+http -a $CLIENT:$SECRET -f $SERVER/auth/token grant_type=client_credentials scope="$SCOPES"
+#| HTTP/1.1 400 Bad Request
+#| 
+#| {
+#|     "error": "invalid_scope",
+#|     "error_description": "The requested scope is invalid, unknown, or malformed."
+#| }
+tail -50 $BASEDIR/spinta.log
+#| ERROR: Authorization server error: invalid_scope: 
+#| Traceback (most recent call last):
+#|   File "authlib/oauth2/rfc6749/authorization_server.py", line 185, in create_token_response
+#|     grant.validate_token_request()
+#|   File "authlib/oauth2/rfc6749/grants/client_credentials.py", line 72, in validate_token_request
+#|     self.validate_requested_scope(client)
+#|   File "authlib/oauth2/rfc6749/grants/base.py", line 92, in validate_requested_scope
+#|     raise InvalidScopeError(state=self.request.state)
+#| authlib.oauth2.rfc6749.errors.InvalidScopeError: invalid_scope: 
+#| INFO: "POST /auth/token HTTP/1.1" 400 Bad Request
```

Comment:
For some reason, I no long can get access token.

Reply:
I have tried to get access token with postman and it worked 
also I have tried with 
windows curl:
`curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d "grant_type=client_credentials&scope=spinta_getone spinta_insert" -u test:secret localhost:8000/auth/token`
it also worked, created auth client with 2 scopes.

then I tried Ubuntu WSL, with Httpie library:
`http -a test:secret -f :8000/auth/token grant_type=client_credentials scope="spinta_getone spinta_getall"`
It also worked, created auth client with 2 scopes.

Tried the same with older version of spinta it also worked there. Spinta seperates scopes with " " spacing, so that could have been the issue.

Reply:
The error appears, when I request `spinta_auth_clients` scope, which is not recognized, but needed for client managment.

Reply:
Created another issue, to show better error message: https://github.com/atviriduomenys/spinta/issues/537.